src/context/AuthContext.js

```diff
@@ -0,0 +1,25 @@
+import { createContext } from "react";
```

Comment:
This file name shouldn't be AuthContext.js , but rather auth.js or someting like that